geomesa-hbase/geomesa-bigtable-dist/src/main/assemblies/component.xml

```diff
@@ -29,7 +29,7 @@
             <outputDirectory>docs</outputDirectory>
         </file>
         <file>
-            <source>../../geomesa-gs-plugin/README.md</source>
+            <source>../../geomesa-archetypes/geomesa-archetypes-gs-plugin/README.md</source>
```

Comment:
``geomesa-archetypes/geomesa-archetypes-gs-plugin/README.md`` needs to be updated too; it still mentions GeoServer 2.8.x.